### ISSUE TITLE
ValueError when trying to run a clustered job

### ISSUE THREAD

**Issue by lbeltrame** [chapmanb/bcbio-nextgen#1254]
cc @LMannarino

This occurred on a ChiP-seq run: latest master code (as of two days ago).

```python
[2016-03-03T08:57Z] uni01: Resource requests: macs2; memory: 1.00; cores: 1
[2016-03-03T08:57Z] uni01: Configuring 1 jobs to run, using 1 cores each with 1.00g of memory reserved for each job
Traceback (most recent call last):
  File "/storage/rozo/bcbio/anaconda/bin/bcbio_nextgen.py", line 226, in <module>
    main(**kwargs)
  File "/storage/rozo/bcbio/anaconda/bin/bcbio_nextgen.py", line 43, in main
    run_main(**kwargs)
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/main.py", line 47, in run_main
    fc_dir, run_info_yaml)
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/main.py", line 82, in _run_toplevel
    for xs in pipeline(config, run_info_yaml, parallel, dirs, samples):
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/main.py", line 327, in chipseqpipeline
    multiplier = peaks._get_multiplier(samples)) as run_parallel:
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/contextlib.py", line 17, in __enter__
    return self.gen.next()
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/bcbio/distributed/prun.py", line 53, in start
    with ipython.create(parallel, dirs, config) as view:
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/contextlib.py", line 17, in __enter__
    return self.gen.next()
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/cluster_helper/cluster.py", line 1022, in cluster_view
    wait_for_all_engines=wait_for_all_engines)
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/cluster_helper/cluster.py", line 906, in __init__
    _start(scheduler, self.profile, queue, num_jobs, cores_per_job, self.cluster_id, extra_params)
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/cluster_helper/cluster.py", line 782, in _start
    resources, specials = _scheduler_resources(scheduler, extra_params, queue)
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/cluster_helper/cluster.py", line 746, in _scheduler_resources
    specials[name] = int(val)
ValueError: invalid literal for int() with base 10: '1.0'
```

No changes were made to the cluster configuration (SLURM) in the mean time.

**Comment by lbeltrame** [comment]
I've seen this also with other jobs: it seems that `val` is a string while it shouldn't be?

**Comment by lbeltrame** [comment]
I see the same error when running local. It looks like the `parallel` object gets a float in `num_cores` for local runs, instead of an int, and joblib complains.

I'm investigating the issue on the clustered mode now.

**Comment by lbeltrame** [comment]
And indeed the issue is the same: `parallel["num_jobs"]` contains a float (9.0, or 1.0, etc.) instead of an int. Float division or anything like that, perhaps?

**Comment by lbeltrame** [comment]
And I just noticed it's been fixed in e4d4d697f183c52fd46d43ea240fbbb9e9d93918. Closing.
